src/main/resources/default_config.xml

```diff
@@ -305,4 +308,4 @@
     </parameters>
   </check>
   <check class="org.scalastyle.scalariform.CurliesImportChecker" level="warning" enabled="false"></check>
-</scalastyle>
+</scalastyle>
```

Comment:
Missing **EOL** before the **EOF**.

Reply:
For some weird reason, this is the default in VSCode 🤔 Just fixed it.